### ISSUE TITLE
Renaming subtask causes text to duplicate

### ISSUE THREAD

**Issue by leonstr** [getting-things-gnome/gtg#450]
Steps to reproduce:

1. In the main window click '+' (Create a New Task).

2. Type "Test task" as the title and press enter.

3. Click "Insert a subtask in this task" on the tool bar and type, e.g. "Subtask 1", and press enter.

4. Go move the caret to the subtask text and change this, e.g. append an 'a' (i.e. "Subtask 1a").
![1-subtask_renamed](https://user-images.githubusercontent.com/10348217/90959165-29267b00-e491-11ea-965c-8eb1a42fd646.png)

5. Close and re-open the task window.

6. The subtask is now listed twice (only once as an actual hyperlinked subtask).
![2-window_reopened](https://user-images.githubusercontent.com/10348217/90959166-2c216b80-e491-11ea-8397-6d85409fa88c.png)

Environment: GTG 0.4.0 on Fedora 29.

**Comment by diegogangl** [comment]
Ah! Saw it this time. It's really tricky to trigger though. What I find weirder is that your screen is not showing the "obj" character.
This is actually written into the XML so it's something affecting the serializer.

**Comment by diegogangl** [comment]
Not happening anymore in Master
